name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/exchange-labels.properties

```diff
@@ -8,6 +8,89 @@
 # 01/2021
 #----------------------------------------------------------------
 
+leeway.AS     = Euronext Amsterdam -> EUR
+leeway.AT     = Athens Exchange -> EUR
+leeway.AU     = Australia Exchange -> AUD
+leeway.BA     = Buenos Aires Exchange -> ARS
+leeway.BC     = Casablanca Stock Exchange -> MAD
+leeway.BE     = Berlin Exchange -> EUR
+leeway.BK     = Thailand Exchange -> THB
+leeway.BR     = Euronext Brussels -> EUR
+leeway.BRVM   = Regional Securities Exchange -> XOF
+leeway.BSE    = Bombay Exchange -> INR
+leeway.BUD    = Budapest Stock Exchange -> HUF
+leeway.CC     = Cryptocurrencies -> Unknown
+leeway.CM     = Colombo Stock Exchange -> LKR
+leeway.CO     = Copenhagen Exchange -> DKK
+leeway.DSE    = Dar es Salaam Stock Exchange -> TZS
+leeway.DU     = Dusseldorf Exchange -> EUR
+leeway.EGX    = Egyptian Exchange -> EGP
+leeway.EUFUND = Europe Fund Virtual Exchange -> EUR
+leeway.F      = Frankfurt Exchange -> EUR
+leeway.FOREX  = FOREX -> Unknown
+leeway.GSE    = Ghana Stock Exchange -> GHS
+leeway.HA     = Hanover Exchange -> EUR
+leeway.HE     = Helsinki Exchange -> EUR
+leeway.HK     = Hong Kong Exchange -> HKD
+leeway.HM     = Hamburg Exchange -> EUR
+leeway.IC     = Iceland Exchange -> ISK
+leeway.IL     = London IL -> USD
+leeway.IR     = Irish Exchange -> EUR
+leeway.IS     = Istanbul Stock Exchange -> TRY
+leeway.JK     = Jakarta Exchange -> IDR
+leeway.KAR    = Karachi Stock Exchange -> PKR
+leeway.KLSE   = Kuala Lumpur Exchange -> MYR
+leeway.KO     = Korea Stock Exchange -> KRW
+leeway.KQ     = KOSDAQ -> KRW
+leeway.LIM    = Bolsa de Valores de Lima -> PEN
+leeway.LS     = Euronext Lisbon -> EUR
+leeway.LSE    = NASDAQ Exchange -> USD
+leeway.LU     = Luxembourg Stock Exchange -> EUR
+leeway.LUSE   = Lusaka Stock Exchange -> ZMW
+leeway.MC     = Madrid Exchange -> CAD
+leeway.MCX    = MICEX Moscow Russia -> RUB
+leeway.MONEY  = Money Market Virtual Exchange -> Unknown
```

Comment:
I propose to remove the "--> Unknown" label.
Just looking at it it reads like the Money Market Virtual Exchange is unknown - (but it can handle multiple currencies)

And then: I am not sure that it is required that all securities are traded at one stock exchange are traded in the same currency.

A quick search brought up this example:
> The Singapore Exchange lists equities denominated in different currencies, including Singapore dollars (SGD), US dollars (USD), Hong Kong dollars (HKD), Renminbi (CNY), Japanese Yen (JPY), Euros (EUR), Australian dollars (AUD), Pound sterling (GBP).
https://economics.stackexchange.com/questions/5399/does-each-stock-exchange-only-trade-in-one-currency


Also:
> leeway.MC     = Madrid Exchange -> CAD
Madrid is trading in Canadian Dollars?

> leeway.NASDAQ = London Exchange -> GBP
> leeway.LSE    = NASDAQ Exchange -> USD
Let's also check the stock exchange labels - I would think London is LSX https://en.wikipedia.org/wiki/London_Stock_Exchange and Nasdaq is NASDAQ